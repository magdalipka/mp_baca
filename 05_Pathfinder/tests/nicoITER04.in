10 10
1 0 1 0 0 1 0 1 0 0
0 0 1 0 1 0 0 0 1 0
0 1 0 0 1 0 1 0 0 0
0 0 1 0 0 0 1 0 1 1
1 0 0 0 1 0 1 0 1 0
0 0 1 1 0 0 1 0 1 0
0 1 0 0 0 1 1 0 0 0
1 1 0 1 1 1 0 0 1 0
0 0 0 1 0 0 0 1 1 0
0 1 1 1 1 1 1 0 0 0
1
i 1 9 8 9

