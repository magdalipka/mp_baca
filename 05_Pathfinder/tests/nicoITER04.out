i S W S S E S E E N E E N N E E S E E N N W
